instrumentation/liberty/liberty/javaagent/liberty-javaagent.gradle

```diff
@@ -0,0 +1,8 @@
+apply from: "$rootDir/gradle/instrumentation.gradle"
+
+dependencies {
```

Comment:
can you add a comment in this file about this library not being available via maven dependency, which is why we need the compile-stub module, and we can't do muzzle library version range validation, and we can't do normal tests and rely on smoke tests for testing?

(and copy-paste or reference the comment from liberty-dispatcher-javaagent.gradle)

Reply:
@laurit Can you also document this stub approach in https://github.com/open-telemetry/opentelemetry-java-instrumentation/blob/master/docs/contributing/writing-instrumentation.md ? As a separate PR, if you wish.

Reply:
doc for using stubs added